Sources/File.swift

```diff
@@ -114,45 +114,45 @@ public final class File: Stream {
 }
 
 extension File {
-    public func write(data: Data, flush: Bool = true, timingOut deadline: Double = .never) throws {
+    public func write(_ data: Data, flush: Bool = true, timingOut deadline: Double = .never) throws {
         try assertNotClosed()
 
         let bytesProcessed = data.withUnsafeBufferPointer {
             filewrite(file, $0.baseAddress, $0.count, deadline.int64milliseconds)
         }
 
-        try FileError.assertNoSendErrorWithData(data, bytesProcessed: bytesProcessed)
+        try FileError.assertNoSendError(with: data, bytesProcessed: bytesProcessed)
 
         if flush {
             try self.flush(timingOut: deadline)
         }
 	}
 
-    public func read(length length: Int, timingOut deadline: Double = .never) throws -> Data {
+    public func read(length: Int, timingOut deadline: Double = .never) throws -> Data {
         try assertNotClosed()
 
         var data = Data.buffer(with: length)
         let bytesProcessed = data.withUnsafeMutableBufferPointer {
             fileread(file, $0.baseAddress, $0.count, deadline.int64milliseconds)
         }
 
-        try FileError.assertNoReceiveErrorWithData(data, bytesProcessed: bytesProcessed)
+        try FileError.assertNoReceiveError(with: data, bytesProcessed: bytesProcessed)
         return Data(data.prefix(bytesProcessed))
     }
 
-    public func read(lowWaterMark lowWaterMark: Int, highWaterMark: Int, timingOut deadline: Double = .never) throws -> Data {
+    public func read(lowWaterMark: Int, highWaterMark: Int, timingOut deadline: Double = .never) throws -> Data {
         try assertNotClosed()
 
         var data = Data.buffer(with: highWaterMark)
         let bytesProcessed = data.withUnsafeMutableBufferPointer {
             filereadlh(file, $0.baseAddress, lowWaterMark, highWaterMark, deadline.int64milliseconds)
         }
 
-        try FileError.assertNoReceiveErrorWithData(data, bytesProcessed: bytesProcessed)
+        try FileError.assertNoReceiveError(with: data, bytesProcessed: bytesProcessed)
         return Data(data.prefix(bytesProcessed))
     }
 
-    public func read(deadline deadline: Double = .never) throws -> Data {
+    public func read(_ deadline: Double = .never) throws -> Data {
```

Comment:
Underscore here is not needed, I think. `read(5.seconds.fromNow())` looks really strange.